hooks/pre/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,55 @@
+def fail_and_exit(message)
+  Kafo::Helpers.log_and_say :error, message
+  kafo.class.exit 1
+end
+
+def migration
+  export_dir = '/var/tmp/engine_upgrade'
```

Comment:
I'd name it `mongodb_engine_upgrade` to be very explicit.

Reply:
Done